llvm/lib/CodeGen/CMakeLists.txt

```diff
@@ -211,6 +211,7 @@ add_llvm_component_library(LLVMCodeGen
   ProfileData
   Scalar
   Support
+  SYCLLowerIR
```

Comment:
Why do we link `SYCLLowerIR` here?
According to my understanding it's not used in llvm's CodeGen.

Reply:
Sorry, I forgot to revert this chenge after implementation improvement. Removed.